integration-tests/api/src/services/local-file-service.js

```diff
@@ -15,49 +15,56 @@ export default class LocalFileService extends AbstractFileService {
     }
   }
 
-  async upload(file) {
-    const uploadPath = path.join(
-      this.upload_dir_,
-      path.dirname(file.originalname)
-    )
+  upload(file) {
```

Comment:
We should keep the `async` for these methods returning a Promise

Reply:
Suggestion: Using [fs Promises](https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api) might be a good fit here

Reply:
yes you are right, I ve been lazy here since it is only for the integration tests 😂

Reply:
In that case that's alright to me. I would just add the `async`

Reply:
I did the cleanup in the end 😂